redux/warGameSlice.test.ts

```diff
@@ -268,3 +269,41 @@ test("test if game is over", () => {
     playerTwoScoring: 0,
   });
 });
+
+test("test reset state", () => {
+  const firstState = reducer(initialState, setPlayerReady(1));
+  const secondState = reducer(firstState, setPlayerReady(2));
+  const thirdState = reducer(secondState, startRound(null));
+  const fourthState = reducer(thirdState, finishRound(1));
+  const fifthState = reducer(fourthState, setPlayerReady(1));
+  const sixthState = reducer(fifthState, setPlayerReady(2));
+  const seventhState = reducer(sixthState, startRound(null));
+  const eighthState = reducer(seventhState, finishRound(1));
+  const ninthState = reducer(eighthState, setPlayerReady(1));
+  const tenthState = reducer(ninthState, setPlayerReady(2));
+  const eleventhState = reducer(tenthState, startRound(null));
+  const twelvethState = reducer(eleventhState, finishRound(1));
+  const thirteenthState = reducer(twelvethState, finishRound(1));
+  expect(reducer(thirteenthState, resetRound("RESET"))).toEqual({
```

Comment:
why do you have to pass this string into resetRound? why can't we just pass in null?

Reply:
We can, I thought to do a condition for reset, but for now we can also pass in Null, will make this change in new commit